shaptools/netweaver.py

```diff
@@ -33,14 +34,17 @@ class NetweaverInstance(object):
         sid (str): SAP Netweaver sid
         inst (str): SAP Netweaver instance number
         password (str): Netweaver instance password
+        remote_host (str, opt): Remote host where the command will be executed
     """
 
     # SID is usualy written uppercased, but the OS user is always created lower case.
     NETWEAVER_USER = '{sid}adm'.lower()
     UNINSTALL_PRODUCT = 'NW_Uninstall:GENERIC.IND.PD'
     GETPROCESSLIST_SUCCESS_CODES = [0, 3, 4]
+    SUCCESSFULLY_INSTALLED = 0
+    UNSPECIFIED_ERROR = 111
```

Comment:
I will confirm the error return codes, we may say everything other than success codes denotes an error

Reply:
@Simranpal, These error codes are retrieved from the `help` output of the `sapinst` installed. I hope they are the correct ones.
Besides that, as we need to manage some errors in a specific way (ASCS manual restart requested for example) is good to have this error code stored in a constant in the code.